node/node.go

```diff
@@ -1613,6 +1620,15 @@ func (app *App) setupDBs(ctx context.Context, lg log.Log) error {
 		datastore.WithConfig(app.Config.Cache),
 		datastore.WithConsensusCache(data),
 	)
+	localDb, err := sql.Open("file:"+filepath.Join(dbPath, localDbFile),
+		sql.WithMigrations(sql.LocalMigrations),
+		sql.WithConnections(app.Config.DatabaseConnections),
+		// TODO(mafa): add data migration to DB here
```

Comment:
Not sure if having a `sql.WithV5Migration` here would be better than just having the to `MoveToDB` methods on the `activation.Builder`?